server/command/list.go

```diff
@@ -4,13 +4,68 @@
 package command
 
 import (
+	"fmt"
+
 	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
 )
 
 func (s *service) executeList(params *params) (*model.CommandResponse, error) {
-	_, txt, err := s.api.Admin.ListApps()
-	if err != nil {
-		return errorOut(params, err)
+	marketplaceApps := s.admin.ListMarketplaceApps("")
+	installedApps := s.admin.ListInstalledApps()
+
+	txt := md.MD("| Name | Status | Version | Account | Locations | Permissions |\n")
+	txt += md.MD("| :-- |:-- | :-- | :-- | :-- | :-- |\n")
+
+	for _, app := range installedApps {
+		mapp := marketplaceApps[app.AppID]
+		if mapp == nil {
```

Comment:
Nit pick:
```suggestion
		mapp, ok := marketplaceApps[app.AppID]
		if !ok || mapp == nil {
```

Reply:
It is unnecessary to use an ok here since `marketplaceApps` is guaranteed to be non-nil.